extensions/rain-radars/src/components/list-empty-view.tsx

```diff
@@ -0,0 +1,18 @@
+import { Grid, List } from "@raycast/api";
+
+export function ListEmptyView(props: { layout: string }) {
+  const { layout } = props;
+  return layout === "List" ? (
```

Comment:
since you're always passing `grid` as `layout`, you can remove the layout prop altogether and just return a `List.EmptyView` here 😸